paasta_tools/api/api_docs/swagger.json

```diff
@@ -96,6 +96,89 @@
           }
         ]
       }
+    },
+    "/services/{service}/{instance}/autoscaler": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Desired instance count for a service instance",
+            "schema": {
+              "format": "integer"
+            }
+          },
+          "404": {
+            "description": "Deployment key not found"
+          },
+          "500": {
+            "description": "Instance failure"
+          }
+        },
+        "summary": "Get status of service_name.instance_name",
+        "operationId": "get_autoscaler_count",
+        "tags": ["autoscaler"],
```

Comment:
You may want to add the autoscaler tag to https://github.com/Yelp/paasta/pull/740/files#diff-529d3a317d236811b355ae5787b6657cL12.